workflow/rocoto/tasks.py

```diff
@@ -131,9 +131,16 @@ def _get_forecast_hours(cdump, config, component='atmos') -> List[str]:
         # Ocean/Ice components do not have a HF output option like the atmosphere
         if component in ['ocean', 'ice']:
             local_config['FHMAX_HF_GFS'] = config['FHMAX_GFS']
```

Comment:
Can probably just make this 0:
```suggestion
            local_config['FHMAX_HF_GFS'] = 0
```

Reply:
Sure, modification has been made.